packages/persist/lib/middleware/auth.middleware.ts

```diff
@@ -25,11 +26,11 @@ export const authMiddleware = (req: Request, res: Response, next: NextFunction)
         .getAccountAndEnvironmentBySecretKey(secret)
         .then((result) => {
             if (!result || result.environment.id !== environmentId) {
-                throw new Error('Unauthorized');
+                throw new Error('Unauthorized: cannot find matching environment');
             }
             next();
         })
-        .catch(() => {
-            res.status(401).json({ error: 'Unauthorized' });
+        .catch((error: unknown) => {
+            res.status(401).json({ error: `Unauthorized ${stringifyError(error)}` });
```

Comment:
it's a detail but it will print `Unauthorized: Unauthorized`

Reply:
good point. fixed with `res.status(401).json({ error: `Auth error: ${stringifyError(error)}` });`